### PR TITLE
New package: ROCm-SMI-6.2.2

### PR DESCRIPTION
**Pull request by Calandracas606**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-musl
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - i686
  - aarch64-musl (cross)
  - armv7l (cross)
  - armv6l-musl (cross)

### CODE REVIEW

File: srcpkgs/ROCm-SMI-LIB/patches/0001-cmake-system-processor.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b3eff5d..defc6b2 100755
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -76,7 +76,7 @@ endif()
+ ## Compiler flags
+ set(CMAKE_CXX_FLAGS
+  "${CMAKE_CXX_FLAGS} -Wall -Wextra -fno-rtti")
+-if (${CMAKE_HOST_SYSTEM_PROCESSOR} STREQUAL "x86_64")
++if (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64")
```
Comment (Calandracas606) [inline srcpkgs/ROCm-SMI-LIB/patches/0001-cmake-system-processor.patch:10]: Fixes cross compiling on arm

Also fixes cross compiling on armv* in the extremely unlikely event that anybody will want to use this package there.

Made a PR upstream: https://github.com/ROCm/rocm_smi_lib/pull/158

**Comment by zen0bit** [comment]
Works for me

**Review by zen0bit — APPROVED** [review]

**Comment by Calandracas606** [comment]
Patch in this PR has been merged upstream:
https://github.com/ROCm/rocm_smi_lib/commit/38d1275d64f06800f3cf46ce198c4ab0cd6bbf42

**Comment by zen0bit** [comment]
bump

**Comment by Calandracas606** [comment]
I'll update this PR to the latest release soon

**Pull request merged**